wagtail/admin/views/workflows.py

```diff
@@ -192,6 +247,17 @@ class TaskIndex(IndexView):
     add_item_label = _("Add a task")
     header_icon = 'placeholder'
 
+    def get_queryset(self):
+        queryset = super().get_queryset()
+        if not strtobool(self.request.GET.get('show_disabled', 'False')):
```

Comment:
I'm not really keen on capitalised parameters in URLs, and importing `strtobool` from distutils is a bit leftfield :-) The standard elsewhere for boolean URL parameters is for the presence of a parameter (with any value) to mean true, so this becomes `if not self.request.GET.get('show_disabled')`.

Reply:
Oops, sorry about that, didn't know the convention for that - I've amended now to just checking for the key in `request.GET`